modular_skyrat/modules/power/code/powerator.dm

```diff
@@ -158,7 +163,10 @@
 	attached_cable.add_delayedload(current_power)
 
 	var/money_ratio = round(current_power * divide_ratio)
-	var/datum/bank_account/synced_bank_account = SSeconomy.get_dep_account(ACCOUNT_CAR)
+	//BUBBER EDIT CHANGE BEGIN - Use credits_account variable for our department look up
+	//var/datum/bank_account/synced_bank_account = SSeconomy.get_dep_account(ACCOUNT_CAR) - BUBBER EDIT - ORIGINAL
+	var/datum/bank_account/synced_bank_account = SSeconomy.get_dep_account(credits_account != "" ? credits_account : ACCOUNT_CAR)
```

Comment:
I do not trust DMs compiler to actually optimize this correctly so let's do this instead
```suggestion
	var/datum/bank_account/synced_bank_account = SSeconomy.get_dep_account(credits_account == "" ? 
 ACCOUNT_CAR : credits_account)
```